archivebox/parsers/twint_json.py

```diff
@@ -14,12 +15,14 @@
 @enforce_types
 def parse_twint_json_export(json_file: IO[str], **_kwargs) -> Iterable[Link]:
     """Parse Twint export using JSON  output"""
-    date_format = "%Y-%m-%d %H:%M:%S%z"
+
     json_file.seek(0)
-    json_date = lambda s: datetime.strptime(s, date_format)
+    json_date = lambda s: datetime.strptime(s, '%Y-%m-%d %H:%M:%S %Z')
+    reader = jsonlines.Reader(json_file)
 
-    for line in json_file:
-        data = json.loads(line)
+    for data in reader.iter(type=dict, skip_invalid=True):
+        print(data)
```

Comment:
```suggestion
        #print(data)
```

I think this was left here from debugging / dev?